src/embed/ts-embed.ts

```diff
@@ -352,6 +353,9 @@ export class TsEmbed {
                 hiddenHomeLeftNavItems: this.viewConfig?.hiddenHomeLeftNavItems
                     ? this.viewConfig?.hiddenHomeLeftNavItems
                     : [],
+                searchOptions: (this.viewConfig as SearchViewConfig).excludeSearchTokenStringFromURL
```

Comment:
`searchOptions` is only specific to SearchEmbed, let's not cross pollute this class. Instead create a generic field here, or create a method `getInitParams()` which returns a null for this class, but the overrides in SearchEmbed adds this field